TeamCharles/initial-site
# 81-product-validation-messages :: added validation to Create and Edit

## Status

**READY**
## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.
## Description

Overload the Create and Edit methods on the ProductsController so that the view can be re-displayed if the model is invalid.
## Resolves Issue Number

Resolves #81 
## Deploy Notes

``` sh
git pull --prune
git checkout 81-product-validation-messages
dotnet run
```
## Steps to Test or Reproduce
1. Log in
2. Click `Sell a Product`
3. Mess up the form, you should be routed back to the form with validation messages
4. Edit an existing product
5. Mess up the form, you should be routed back to the form with validation messages
6. Make sure you can actually complete adding a new product and committing edits to an existing product
## Impacted Files in Application

List general components of the application that this PR will affect:
- Views/Products/Create
- Views/Products/Edit
- Controllers/ProductsController
